transformations/space_between_characters/README.md

```diff
@@ -0,0 +1,20 @@
+# Space Between Characters
+This perturbation adds noise to all types of text sources (sentence, paragraph, etc.).
+
+Author name: Marco Di Giovanni
+Author email: marco.digiovanni@polimi.it
+Author Affiliation: Politecnico di Milano and University of Bologna
+
+## What type of a transformation is this?
+This transformation acts like a perturbation to test robustness. Few words are picked at random and spaces are added between characters (e.g., Marco -> M a r c o)
+
+Generated transformations display high similarity to the source sentences i.e. the code outputs highly precise and readable generations.
+
+## What tasks does it intend to benefit?
+This perturbation would benefit all tasks which have a sentence/paragraph/document as input like text classification, text generation, etc.
+
+
+## What are the limitations of this transformation?
+- The transformation's outputs are very simple.
+- It is not capable of generating linguistically diverse text.
+- This transformation will mainly affect the perfornamce of token/word-level models, while character-level models should be much robust.
```

Comment:
nit: "much more robust"

Reply:
Thank you @sebastianGehrmann for your suggestion. I agree that it is very interesting to expand this transformation by adding the possibility of not having a space. I have implemented it in b551a5c1dea6ff0a3d8095dbb76cfad1f79f80b4 where I added a new argument controlling the probability of inserting a space between 2 characters in a token. 
I have also updated the README in 28b1301c23bd7105017b274a64501f32328309f1